packages/vue-components/src/Thumbnail.vue

```diff
@@ -1,20 +1,22 @@
 <template>
-  <div
-    :class="['thumb-wrapper', {'thumb-circle': circle}, addClass]"
-    :style="[getBorder, getFontSize, getFontColor, getBgColor]"
-  >
-    <img
-      v-if="hasSrc"
-      ref="pic"
-      :src="src"
-      class="thumb-image"
-      :width="computedWidth"
-      :height="computedHeight"
-      :alt="alt"
-      @load.once="computeImgSize"
-    />
-    <slot></slot>
-  </div>
+  <span>
```

Comment:
> Hmm I was thinking that making the component changes would be appropriate in this PR as well. Do you think we can keep the changes in this PR? Regardless, I've made the changes to check if they are what you were talking about. :-)

Hmm I'm thinking the SSR pr might be better here as it allows us to actually test whether this is SSR-compliant.

E.g. over here there's a `span` within a `div`, which is likely to cause issues.

Reply:
Hmm I see where you're coming from. In that case, yeap I agree as well!